Dockerfile

```diff
@@ -7,6 +7,7 @@ COPY ./public ./public
 COPY tsconfig.json next.config.js next-env.d.ts tailwind.config.js postcss.config.js convert-theme-to-ts theme.js ./
 
 ARG NEXT_PUBLIC_GIT_HASH=unknown
+ARG NEXT_PUBLIC_NODE_ENV=production
```

Comment:
this should be documented to Docker Reference as it is most likely going to be used in the docker-compose environment